web/src/click_handlers.js

```diff
@@ -641,6 +642,7 @@ export function initialize() {
 
     user_profile.register_click_handlers();
     stream_popover.register_click_handlers();
+    sidebar_ui.register_click_handlers();
```

Comment:
So we're trying to remove these initialization calls out of this file -- just register the click handlers in an `initialize` method called from `ui_init.js` like we do other modules.

Reply:
(And in fact a pair of commits to rename `stream_popover.register_click_handlers` to `initialize` and call it from `ui_init`, and similarly for `user_profile.js`, would be great)

Reply:
Ok, done.